### ISSUE TITLE
ipfs.object.put error

### ISSUE THREAD

**Issue by kenshyx** [ipfs-shipyard/ipfs-companion#449]
Because of https://github.com/tableflip/ipfs-postmsg-proxy package, if I'm trying to add an object using the provided `window.ipfs` instance ex: `ipfs.object.put({a:1}, (err, result) => console.log(err, result))` it will always fail with error: 

`Uncaught TypeError: _ipldDagPb.DAGNode.isDAGNode is not a function`.

**Comment by lidel** [comment]
I merged #448 which fixes issue with `isDAGNode` and released it as [v2.2.1beta1](https://github.com/ipfs-shipyard/ipfs-companion/releases/tag/v2.2.1beta1). I'll release it to stable channel shortly.

@kenshyx Note that your snippet will still fail, as  the object is not valid (missing `Data` field – see [spec](https://github.com/ipfs/interface-ipfs-core/blob/master/SPEC/OBJECT.md#objectput)). 
Try this instead:

```js
ipfs.object.put({ Data: Buffer.from('Some data'), Links: [] }, (err, node) => {
  if (err) {
    throw err
  }
  console.log(node.toJSON().multihash)
  // Logs:
  // QmPb5f92FxKPYdT3QNBd1GKiL4tZUXUrzF4Hkpdr3Gf1gK
})
```

**Comment by lidel** [comment]
Fixed in [v2.2.1](https://github.com/ipfs-shipyard/ipfs-companion/releases/tag/v2.2.1)
